pages/community.vue

```diff
@@ -6,30 +6,36 @@
           class="font-extrabold text-2xl max-w-lg"
         >We're on Discord!</h1>
       </header>
-      <div class="py-2">
-        <a
-          href="https://discord.gg/GCmCgcM"
-          class="font-semibold border border-primary text-primary px-4 py-1 hover:shadow"
-          :to="`/events/${slug}`"
-        >Join the Discord Channel</a>
-      </div>
-      <div class="w-full flex flex-row justify-center items-center py-12">
-        <iframe
-          src="https://discordapp.com/widget?id=714103890093801512&theme=dark"
-          width="350"
-          height="500"
-          allowtransparency="true"
-          frameborder="0"
-        ></iframe>
-      </div>
+      <main>
+        <div class="py-2">
+          <a
+            href="https://discord.gg/GCmCgcM"
+            class="font-semibold border border-primary text-primary px-4 py-1 hover:shadow"
+            :to="`/events/${slug}`"
+          >Join the Discord Channel</a>
+        </div>
+        <div class="w-full flex flex-row justify-center items-center py-12">
+          <iframe
+            src="https://discordapp.com/widget?id=714103890093801512&theme=dark"
+            height="500"
+            allowtransparency="true"
+            frameborder="0"
+            title="Discord Widget"
+          ></iframe>
+        </div>
+      </main>
     </section>
   </div>
 </template>
 
 <style lang="postcss">
 h1 {
-  background: linear-gradient(114.63deg, #ffb99b 47.46%, #ffccb6 76.87%);
+  background: linear-gradient(114.63deg, #A35F3E 47.46%, #C17E5C 76.87%);
```

Comment:
Should probably be pulling all these color values from the tailwind config

Reply:
is there a good way in Vue to use JS vars in the style tag?

Reply:
‘:style=‘
Can pass an array or use string interpolation

Reply:
updated the remaining gradient to pull values from the config in index.vue but ran into some issues with the specificity of inline styles. the background property as an inline style is too specific and overriding background-clip, but it doesn't look like Vue supports style binding for background-clip. ended up using an !important in the scoped component style which is definitely less than ideal. a better approach might be storing the entire gradient string in the config directly so that it can be applied with a tailwind class, but it's late